src/controllers/simplified-account/settings/service-name/edit/edit-service-name.controller.test.ts

```diff
@@ -0,0 +1,236 @@
+import ControllerTestBuilder from '@test/test-helpers/simplified-account/controllers/ControllerTestBuilder.class'
```

Comment:
I had converted this test file into Typescript for the other PR, so I am adding it here not to waste it.